corehq/warehouse/transforms/sql/application_dim.sql

```diff
@@ -0,0 +1,23 @@
+INSERT INTO {{ application_dim }} (
+    application_id,
+    domain,
+    name,
+    application_last_modified,
+    deleted,
+    dim_last_modified,
+    dim_created_on,
+    batch_id
+)
+SELECT
+    application_id,
+    domain,
+    name,
+    application_last_modified,
+    CASE base_doc
+        WHEN 'ApplicationBase' then false
+        WHEN 'ApplicationBase-Deleted' then true
+    END,
+    now(),
+    now(),
```

Comment:
I followed the pattern of the other sql of making both `dim_last_modified` and `dim_created_on` equal to `now()` but i am not sure why `created_on` would be